app/components/admin/impersonation_banner_component.html.erb

```diff
@@ -0,0 +1,7 @@
+<% if user&.dfe_user_impersonating_school_user? %>
+  <p class='govuk-body'>
+    <%= "You are signed in as #{@school&.name}" %>
+  </p>
+
+  <%= govuk_button_to("Sign out from #{@school.name}", admin_impersonate_path, method: :delete) %>
```

Comment:
Thoughts on making the sign in and sign out buttons secondary? Finding it a bit distracting 😅
```suggestion
  <%= govuk_button_to("Sign out from #{@school.name}", admin_impersonate_path, method: :delete, class: "govuk-button--secondary") %>
```

Reply:
Fixed this now, see the screenshot above.